app/plans/displayText.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  heading: 'Active plan',
+  start_plan: 'Start a new plan',
+  no_plan: 'There is no active plan',
+}
```

Comment:
What do we gain from this? They're referenced in the nunjucks file with the rest of the text, so the page text ends up split over two files?

Reply:
Integrated text into njk file